src/deepsparse/v2/operators/engine_operator.py

```diff
@@ -142,7 +143,8 @@ def create_engine(
             constructor/compilation
         :return: inference engine
         """
-        onnx_file_path = self.model_path
+
+        onnx_file_path = kwargs.pop("model_path", self.model_path)
```

Comment:
Why do we need to pop?

Reply:
`model_path` isn't a kwarg needed for engine creation.  alternatively, could use the correct name and use `kwargs.get(...)` to handle the case where it isnt provided.